airflow/providers/google/cloud/example_dags/example_dlp.py

```diff
@@ -66,20 +68,124 @@
         do_xcom_push=True,
         dag=dag,
     )
+    # [END howto_operator_dlp_create_inspect_template]
 
+    # [START howto_operator_dlp_use_inspect_template]
     inspect_content = CloudDLPInspectContentOperator(
         task_id="inpsect_content",
         project_id=GCP_PROJECT,
         item=ITEM,
         inspect_template_name="{{ task_instance.xcom_pull('create_template', key='return_value')['name'] }}",
         dag=dag,
     )
+    # [END howto_operator_dlp_use_inspect_template]
 
+    # [START howto_operator_dlp_delete_inspect_template]
     delete_template = CloudDLPDeleteInspectTemplateOperator(
         task_id="delete_template",
         template_id=TEMPLATE_ID,
         project_id=GCP_PROJECT,
         dag=dag,
     )
+    # [END howto_operator_dlp_delete_inspect_template]
 
     create_template >> inspect_content >> delete_template
+
+
+custom_info_types = [
+    {
+        "info_type": {"name": "C_MRN"},
+        "regex": {"pattern": "[1-9]{3}-[1-9]{1}-[1-9]{5}"},
+    }
+]
+custom_info_type_id = "custom_info_type"
+update_custom_info_type = [
+    {
+        "info_type": {"name": "C_MRN"},
+        "regex": {"pattern": "[a-z]{3}-[a-z]{1}-[a-z]{5}"},
+    }
+]
+
+with models.DAG(
+    "example_gcp_dlp_info_types",
+    default_args=default_args,
+    schedule_interval=None,
+    tags=["example", "dlp", "info-types"],
+) as dag:
+    # [START howto_operator_dlp_create_info_type]
+    create_info_type = CloudDLPCreateStoredInfoTypeOperator(
+        project_id=GCP_PROJECT,
+        config=custom_info_types,
+        stored_info_type_id=custom_info_type_id,
+        dag=dag,
+        task_id="create_info_type",
+    )
```

Comment:
This task fails with:
```
  File "/airflow/airflow/providers/google/cloud/hooks/dlp.py", line 432, in create_stored_info_type
    metadata=metadata,
  File ".virtualenvs/airflow/lib/python3.6/site-packages/google/cloud/dlp_v2/gapic/dlp_service_client.py", line 2625, in create_stored_info_type
    parent=parent, config=config, stored_info_type_id=stored_info_type_id
TypeError: Parameter to MergeFrom() must be instance of same class: expected google.privacy.dlp.v2.StoredInfoTypeConfig got list.
```

Reply:
I'm happy to help with solving this but probably it would be best if the same person who develops the DAG run the DAG to verify if the example works. @mik-laj @kaxil

Reply:
I agree